### ISSUE TITLE
:boom: I blew up the bell!

### ISSUE THREAD

**Issue by WendyStarfall** [OpenCollar/opencollar#614]
```
Too many sound requests.  Throttled until average falls.
Too many sound requests.  Throttled until average falls.
Too many sound requests.  Throttled until average falls.
Too many sound requests.  Throttled until average falls.
Too many sound requests.  Throttled until average falls.
Too many sound requests.  Throttled until average falls.
Too many sound requests.  Throttled until average falls.
Too many sound requests.  Throttled until average falls.
Too many errors... dropping further messages until the flood stops
```

I could provoke this by double-tapping my forward arrow or W key to make the avatar run or by toggling run with Ctrl + R and then simply moving forwards. Errors happen after approx. 2 seconds of running. :speak_no_evil:

**Comment by XenHat** [comment]
I noticed that as well today.
Confirmed on BETA I, BETA II and BETA III.
Approximate time before throttle: 3-4 seconds of continuous running with the bell plugin activated.

![hqhnek](https://cloud.githubusercontent.com/assets/980408/8978480/97765adc-3670-11e5-953b-2add168d2210.png)
![image](https://cloud.githubusercontent.com/assets/980408/8978502/c1c99eb6-3670-11e5-9f25-7159dcf31c3e.png)

![image](https://cloud.githubusercontent.com/assets/980408/8978457/68f21778-3670-11e5-802e-0696e8446df7.png)

**Comment by WendyStarfall** [comment]
I will also test what happens if there are a lot of collisions in quick succession.

Finally I have a good reason to randomly bump into random people at random! :paw_prints:

PS: That is a lot harder to do alone than I thought it would be......

**Comment by WendyStarfall** [comment]
Thanks!!

**Comment by GarvinTwine** [comment]
fixed in #616
